cdc/api/owner.go

```diff
@@ -82,11 +82,17 @@ type ownerAPI struct {
 // RegisterOwnerAPIRoutes registers routes for owner APIs.
 func RegisterOwnerAPIRoutes(router *gin.Engine, capture *capture.Capture) {
 	ownerAPI := ownerAPI{capture: capture}
-	router.POST("/capture/owner/resign", gin.WrapF(ownerAPI.handleResignOwner))
-	router.POST("/capture/owner/admin", gin.WrapF(ownerAPI.handleChangefeedAdmin))
-	router.POST("/capture/owner/rebalance_trigger", gin.WrapF(ownerAPI.handleRebalanceTrigger))
-	router.POST("/capture/owner/move_table", gin.WrapF(ownerAPI.handleMoveTable))
-	router.POST("/capture/owner/changefeed/query", gin.WrapF(ownerAPI.handleChangefeedQuery))
+	owner := router.Group("/capture/owner")
+
+	owner.Use(errorHandleMiddleware())
+	owner.Use(logMiddleware())
+	owner.Use(timeoutMiddleware(10))
```

Comment:
Could you add some comments? Why we need 10s timeout? what if we want profile more than 10s? `curl .../debug/pprof/profile?seconds=20`